src/main/java/racingcar/RacingDice.java

```diff
@@ -0,0 +1,21 @@
+package racingcar;
+
+import racingcar.race.Dice;
+
+import java.util.Random;
+
+public class RacingDice implements Dice {
+    public static int RACING_MAX_BOUND = 10;
+    int bound;
+
+    public RacingDice(int bound) {
+        this.bound = bound;
+    }
```

Comment:
다양한 랜덤값 생성에 대비하여 만드신 DIce 구조 좋았습니다 👍 
첨언하자면, RACING_MAX_BOUND만큼의 bound 값을 가지고 생성하는 정적 팩토리 메소드를 만들어서 사용하는건 어떨까요? g
```suggestion
    public RacingDice(int bound) {
        this.bound = bound;
    }
    
    public static RacingDice newInstance() {
        return new RacingDice(RACING_MAX_BOUND);
    }
```
현재 조건에서 가장 많이 쓰이는 bound 값이니, 이렇게 사용해도 좋을 것 같습니다:)

Reply:
보통 정적팩토리 메소드를 사용 할 때 생성자의 접근 제한자를 어떻게 두시나요?
이 경우 public 을 private 로 변경해도 될 것 같은데 어떻게 생각하시나요?